packages/core/content-type-builder/admin/src/components/FormModalNavigationProvider/FormModalNavigationProvider.tsx

```diff
@@ -178,6 +184,7 @@ export const FormModalNavigationProvider = ({ children }: FormModalNavigationPro
         forTarget,
         targetUid,
         modalType: 'chooseAttribute',
+        activeTab: 'basic',
       };
     });
   };
```

Comment:
I think it's only at this point `onNavigateToChooseAttributeModal` that we need
to ensure activeTab is reset to 'basic'?

Reply:
I prefered to always reset it just in case (thinking about when you cancel or go back a step what happens) 
but happy to change if you think it's unnecessary